papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraAuthorisationService.java

```diff
@@ -0,0 +1,82 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthParameters;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.util.Value;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Optional;
+import java.util.Scanner;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.json.JSONArray;
+import org.springframework.stereotype.Service;
+
+@Service
+@Getter
+@Setter
+@NoArgsConstructor
+public class JiraAuthorisationService {
+
+    @Value("${jira.auth.private.key}")
+    private static String PRIVATE_KEY;
```

Comment:
Jest zaimportowane inne `Value` aktualnie, ale jakby było to ze Sprint Boota,  to nie wydaje mi się, żeby `static` miało tu działać - te pola mają ustawioną wartość dopiero *po* konstrukcji obiektu (więc na przykład wewnątrz konstruktora są jeszcze same nulle)

Reply:
Usunięto zbędną/niewłaściwą adnotacje `static`